docker-compose.yml

```diff
@@ -234,6 +234,13 @@ services:
     restart: unless-stopped
     tty: false
 
+  explorer:
+    container_name:  hedera-explorer
+    image: "glazarov95/hedera-explorer:latest"
+    restart: "unless-stopped"
+    ports:
+      - "9090:80"
```

Comment:
There should be a README update surfacing this to first time users.
Not sure if there's a section for each exposed endpoint but if not we can start with one section that just lists all the accessible endpoints

Reply:
Yes, that is a good idea. I will do that .